app/controllers/rounds_controller.rb

```diff
@@ -1,6 +1,26 @@
 class RoundsController < ApplicationController
-  # Start creating a round after Start button has been clicked
+  # Start creating a round after Start button or Next Question button has been clicked
   def create
-    
+    @instance = Instance.find(params[:instance_id])
+
+    if @instance.status != "ongoing"
```

Comment:
If for any reason the status of the instance is `completed`, it would be safer to have `if @instance.status == "waiting"` here.

Reply:
I just changed it! Thanks a thousand for reviewing this huge thing!